CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+# Changelog
```

Comment:
@choeppler  should we include the release notes for previous versions as well ?

Reply:
Yes, I think so -- feel free to add them... ;-)

Reply:
Added previous versions, starting with v0.3.0 in 94b624d2